<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>root</class>
 <widget class="QWidget" name="root">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="selector">
      <attribute name="title">
       <string>Selector</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0">
         <item row="0" column="0">
          <widget class="QComboBox" name="jobs">
           <item>
            <property name="text">
             <string>Select path</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="folders_btn">
           <property name="text">
            <string>Add Folder</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="editBtn">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="subDir"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="explorerBtn">
           <property name="text">
            <string>Open Explorer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="newBtn">
           <property name="text">
            <string>New Empty File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="incBtn">
           <property name="text">
            <string>Save Incremental</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="initBtn">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Initialize Working Folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="flipBtn">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>ðŸŽ¬Flipbook</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hipbookBtn">
           <property name="toolTip">
            <string>Non-dated flipbook called $HIPNAME. Shift Click for custom name.</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>ðŸŽ¬$HIP-Book</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="filterString">
           <property name="placeholderText">
            <string>Prefix filter, eg. &quot;Project_&quot;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="latestToggle">
           <property name="text">
            <string>Latest version only</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table">
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <column>
          <property name="text">
           <string>Project</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Version</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="prefWindow">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="prefGrid">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QLineEdit" name="prjRoot"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Houdini Root</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="pathPrev">
           <property name="text">
            <string>$JOB/</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="rootBtn">
           <property name="text">
            <string>Save Houdini Root</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Notes">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Notes:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="notes"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="infoRow">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
